### PR TITLE
Fix/200 decode and display related guides and stories

### LINKED ISSUE #200: Decode & display related guides and stories
## Acceptance Criteria

- [ ] If a guide has related guides, teasers for them are displayed on the guide
- [ ] If a guide has related stories, teasers for them are displayed on the guide
- [ ] Guide teasers are a list of titles linked to the guide
- [ ] Story teasers are a grid of image with title underneath linking to the story (note the title has changed from the one in the design) - still TBC

image
image

## Implementation plan

- [ ] Alter the decode to get title of related rather than the data
- [ ] Look-up the content from title
- [ ] Display as per designs

### PR DESCRIPTION
**Pull request by katjam**
Fixes #200

## Description

- Display list of related guides
- Display list of related stories (WIP)

@geeksforsocialchange/developers

### CODE REVIEW

File: review/elm.json [hunk 1]
```diff
@@ -1,39 +1,41 @@
 {
-  "type": "application",
```
Comment (katjam) [inline review/elm.json:2]: Not strictly part of this refactor but - Needed to update `elm-review` to run lint

File: src/Page/Guide/View.elm [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
 import Theme.FluidScale exposing (fontSize1)
-import Theme.Global exposing (centerContent, contentWrapper, pageColumnBlockStyle, pageColumnStyle, topTwoColumnsWrapperStyle)
+import Theme.Global exposing (centerContent, contentWrapper, pageColumnBlockStyle, pageColumnStyle, roundedCornerStyle, teaserImageStyle, topTwoColumnsWrapperStyle)
 import Theme.Markdown exposing (markdownToHtml)
 
 
-view : Page.Guide.Data.Guide -> Html Msg
-view guide =
+view : Language -> Page.Guide.Data.Guide -> List Page.Guide.Data.GuideListItem -> List Page.Story.Data.StoryTeaser -> Html Msg
+view language guide allGudes allStories =
```
Comment (aaaaargZombies) [inline src/Page/Guide/View.elm:20]: just a typo in `allGudes`

**Review by aaaaargZombies — APPROVED** [review]
nice, just a tiny nit pick

File: src/Page/Guide/View.elm (hunk 2 above)
Comment (katjam) [reply src/Page/Guide/View.elm:20]: Thanks! I'll fix that.

**Pull request merged**